<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CParameterSSHTunnelUI</class>
 <widget class="QWidget" name="CParameterSSHTunnelUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="CParameterNetUI" name="wNet" native="true"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source host:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSourceHost"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Source port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbSourcePort">
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Packge capture file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePcapFile"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CParameterNetUI</class>
   <extends>QWidget</extends>
   <header location="global">ParameterNetUI.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
